lib/Conversion/TTNNToEmitC/TTNNToEmitC.cpp

```diff
@@ -376,7 +454,7 @@ class ToLayoutOpConversionPattern
     attrs.push_back(convertLayoutAttr(rewriter, srcOp.getLayoutAttr()));
     attrs.push_back(createStdNullopt(rewriter));
     attrs.push_back(createStdNullopt(rewriter));
-    attrs.push_back(mlir::IntegerAttr::get(rewriter.getIndexType(), 1));
+    attrs.push_back(createNullDevicePointer(rewriter));
```

Comment:
Why this change?

Reply:
Because toLayoutOps shouldn't be moving tensors to device, that's the responsibility of toDeviceOps. Therefore we set the device pointer to null.